himbaechel/himbaechel_dbgen/chip.py

```diff
@@ -144,6 +144,7 @@ class TileWireData:
     index: int
     name: IdString
     wire_type: IdString
+    tile_wire: int
```

Comment:
this should be given a name that makes it's very clear that it's gui-specific, otherwise it will be very unclear to anyone looking at the structures